pkg/testframework/imageregistry.go

```diff
@@ -18,9 +18,9 @@ import (
 )
 
 const (
-	ImageRegistryName                = "image-registry"
-	ImageRegistryDeploymentName      = ImageRegistryName
-	ImageRegistryDeploymentNamespace = OperatorDeploymentNamespace
+	ImageRegistryResourceName        = imageregistryapi.ImageRegistryResourceName
+	ImageRegistryDeploymentName      = imageregistryapi.ImageRegistryName
+	ImageRegistryDeploymentNamespace = imageregistryapi.ImageRegistryOperatorNamespace
```

Comment:
not really sure why you created these instead of just using the api constants directly, but i'm not going to hold this up over it since we need it in asap.

Reply:
I basically just updated them, but do agree they can be replaced by the ones in the api in the future